docs/source/nodes_and_pipelines/micro_packaging.md

```diff
@@ -10,7 +10,7 @@ You can package a micro-package by executing: `kedro micropkg package <micropkg_
 
 `kedro micropkg package pipelines.data_processing`
 
-* This will generate a new [source distribution](https://docs.python.org/3/distutils/sourcedist.html) for this micro-package.
+* This will generate a new source distribution for this micro-package.
```

Comment:
I was about to suggest 

```suggestion
* This will generate a new [source distribution](https://packaging.python.org/en/latest/glossary/#term-Source-Distribution-or-sdist) for this micro-package.
```

but that page is not only barely useful, but also outdated... ugh

Reply:
xref https://github.com/pypa/packaging.python.org/pull/1309